packages/theme/src/cli/commands/theme/dev.ts

```diff
@@ -54,6 +54,15 @@ You can run this command only in a directory that matches the [default Shopify t
       options: ['hot-reload', 'full-page', 'off'],
       env: 'SHOPIFY_FLAG_LIVE_RELOAD',
     }),
+    'error-overlay': Flags.string({
+      description: `Controls the visibility of the error overlay when an theme asset upload fails:
+- silent Prevents the error overlay from appearing.
+- default Displays the error overlay.
+      `,
+      options: ['silent', 'default'],
```

Comment:
Instead of passing values, should we pass `allowNo: true` so that devs can use the flag as `--no-error-overlay` to disable the behavior? Or do we expect to have more options in the future?

Reply:
yeah, the idea is to leave room for expansion moving forward, though there's no clear timeline for that so perhaps something we could call into question